Dockerfile

```diff
@@ -14,16 +11,18 @@ RUN if [ -d .git ]; then echo "ERROR: .dockerignore folders detected, exiting" &
 
 # Install and prepare install
 RUN mkdir -p web/var \
-    && composer install --optimize-autoloader --no-progress --no-interaction --no-suggest --prefer-dist \
+    # For now, only run composer in order to generate parameters.yml
+    && composer run-script build --no-interaction \
```

Comment:
should then add `composer dump -o` as well then to make sure autoload is optimized.